### PR TITLE
Specify the difference between classification/profiling

### LINKED ISSUE #211: Clarify taxonomic classification vs profiling everywhere
### Description of the bug

As suggested by @Midnighter https://nfcore.slack.com/archives/C031QH57DSS/p1673863932137969?thread_ts=1673858047.865519&cid=C031QH57DSS

(i.e. classification - who is there, profiling - how much of who is there)

### Command used and terminal output

_No response_

### Relevant files

_No response_

### System information

_No response_

### PR DESCRIPTION
**Pull request by jfy133**
Closes #211

### CODE REVIEW

**Comment by jfy133** [comment]
@nf-core-bot fix linting

File: docs/usage.md [hunk 1]
```diff
@@ -64,9 +64,9 @@ An [example samplesheet](../assets/samplesheet.csv) has been provided with the p
 
 ### Full database sheet
 
-nf-core/taxprofiler supports multiple databases being profiled in parallel for each tool.
+nf-core/taxprofiler supports multiple databases being classified/profiled against in parallel for each tool.
 Databases can be supplied either in the form of a compressed `.tar.gz` archive of a directory containing all relevant database files or the path to a directory on the filesystem.
-The pipeline takes the locations and specific profiling parameters of the tool of these databases as input via a four column comma-separated sheet.
+The pipeline takes the locations and specific classification/profiling parameters of the tool of these databases as input via a four column comma-separated sheet.
```
Comment (sofstam) [inline docs/usage.md:69]:
```suggestion
The pipeline takes the pathways and specific classification/profiling parameters of the tool of these databases as input via a four column comma-separated sheet.
```

File: docs/usage.md [hunk 2]
```diff
@@ -153,9 +153,9 @@ It is highly recommended to run this on raw reads to remove artifacts from seque
...
 You can also turn off clipping and only perform paired-end merging, if requested. This can be useful when processing data downloaded from the ENA, SRA, or DDBJ (`--shortread_qc_skipadaptertrim`).
-Both tools support length filtering of reads and can be tuned with `--shortread_qc_minlength`. Performing length filtering can be useful to remove short (often low sequencing complexity) sequences that result in unspecific classification and therefore slow down runtime during profiling, with minimal gain.
+Both tools support length filtering of reads and can be tuned with `--shortread_qc_minlength`. Performing length filtering can be useful to remove short (often low sequencing complexity) sequences that result in unspecific classification and therefore slow down runtime during classificationprofiling, with minimal gain.
```
Comment (sofstam) [inline docs/usage.md:158]:
```suggestion
Both tools support length filtering of reads and can be tuned with `--shortread_qc_minlength`. Performing length filtering can be useful to remove short (often low sequencing complexity) sequences that result in unspecific classification and therefore slow down runtime during classification/profiling, with minimal gain.
```

File: docs/usage.md [hunk 3]
```diff
@@ -179,11 +179,11 @@ You can optionally save the FASTQ output of the run merging with the `--save_com
...
+Similarly to complexity filtering, host-removal can be useful for runtime optimisation and reduction in misclassified reads. It is not always necessary to report classification of reads from a host when you already know the host of the sample, therefore you can gain a run-time and computational advantage by removing these prior typically resource-heavy classification/profiling with more efficient methods. Furthermore, particularly with human samples, you can reduce the number of false positives during classification/profiling that occur due to host-sequence contamination in reference genomes on public databases.
 
-nf-core/taxprofiler currently offers host-removal via alignment against a reference genome with Bowtie2, and the use of the unaligned reads for downstream profiling.
+nf-core/taxprofiler currently offers host-removal via alignment against a reference genome with Bowtie2, and the use of the unaligned reads for downstream classification/profiling.
```
Comment (sofstam) [inline docs/usage.md:186]:
```suggestion
nf-core/taxprofiler currently offers host-removal via alignment against a reference genome with Bowtie2 for short reads and minimap2 for long reads, and the use of the unaligned reads for downstream classification/profiling.
```

File: docs/usage.md [hunk 4]
```diff
@@ -195,13 +195,19 @@ For samples that may have been sequenced over multiple runs, or for FASTQ files
...
+Activating this functionality will concatenate the FASTQ files with the same sample name _after_ the optional preprocessing steps and _before_ classification/profiling. Note that libraries with runs of different pairing types will **not** be merged and this will be indicated on output files with a `_se` or `_pe` suffix to the sample name accordingly.
 
 You can optionally save the FASTQ output of the run merging with the `--save_runmerged_reads`.
 
-#### Profiling
+#### Classification and Profiling
 
-The following suggestion gives you some tips and suggestions regarding running some of the different tools specifically _within the pipeline_. For advice as to which tool to run in your context, please see the documentation of each tool.
+The following suggestion gives you some tips and suggestions regarding running some of the different tools specifically _within the pipeline_.
```
Comment (sofstam) [inline docs/usage.md:204]: Maybe use some other word other than suggestion since it is used two times in the same sentence?

Comment (jfy133) [inline docs/usage.md:204]:
```suggestion
The following sections gives you some tips and suggestions regarding running some of the different tools specifically _within the pipeline_.
```

**Review by sofstam — APPROVED** [review]

**Pull request merged**